Unable to install on Windows via cargo install ion-cli

Just tried installing this on my Windows machine via cargo install ion-cli, fails to compile with the following errors:

```
   Compiling ion-cli v0.7.0
error[E0004]: non-exhaustive patterns: `ion_rs::MacroExprKind::TemplateArgGroup(_)` not covered
   --> C:\Users\aaron\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ion-cli-0.7.0\src/bin/ion\commands\inspect.rs:449:27
    |
449 |                     match invocation.kind() {
    |                           ^^^^^^^^^^^^^^^^^ pattern `ion_rs::MacroExprKind::TemplateArgGroup(_)` not covered
    |
note: `ion_rs::MacroExprKind<'_, ion_rs::AnyEncoding>` defined here
   --> C:\Users\aaron\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ion-rs-1.0.0-rc.8\src\lazy\expanded\macro_evaluator.rs:155:1
    |
155 | pub enum MacroExprKind<'top, D: Decoder> {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
159 |     TemplateArgGroup(TemplateExprGroup<'top, D>),
    |     ---------------- not covered
    = note: the matched value is of type `ion_rs::MacroExprKind<'_, ion_rs::AnyEncoding>`
help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern or an explicit pattern as shown
    |
454 ~                         },
455 +                         ion_rs::MacroExprKind::TemplateArgGroup(_) => todo!()
    |

error[E0004]: non-exhaustive patterns: `ion_rs::ExpandedSExpSource::Constructed(_, _)` not covered
   --> C:\Users\aaron\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ion-cli-0.7.0\src/bin/ion\commands\inspect.rs:532:15
    |
532 |         match sexp.expanded().source() {
    |               ^^^^^^^^^^^^^^^^^^^^^^^^ pattern `ion_rs::ExpandedSExpSource::Constructed(_, _)` not covered
    |
note: `ion_rs::ExpandedSExpSource<'_, ion_rs::AnyEncoding>` defined here
   --> C:\Users\aaron\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ion-rs-1.0.0-rc.8\src\lazy\expanded\sequence.rs:208:1
    |
208 | pub enum ExpandedSExpSource<'top, D: Decoder> {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
216 |     Constructed(Environment<'top, D>, MacroExprArgsIterator<'top, D>),
    |     ----------- not covered
    = note: the matched value is of type `ion_rs::ExpandedSExpSource<'_, ion_rs::AnyEncoding>`
help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern or an explicit pattern as shown
    |
547 ~             },
548 +             ion_rs::ExpandedSExpSource::Constructed(_, _) => todo!()
    |

For more information about this error, try `rustc --explain E0004`.
error: could not compile `ion-cli` (bin "ion") due to 2 previous errors
error: failed to compile `ion-cli v0.7.0`, intermediate artifacts can be found at `C:\Users\aaron\AppData\Local\Temp\cargo-installtUxYuV`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "ion-cli"
-version = "0.7.0"
+version = "0.7.1"
 authors = ["The Ion Team <ion-team@amazon.com>"]
 edition = "2021"
 description = "Command line tool for working with the Ion data format."
@@ -20,7 +20,7 @@ sha2 = "0.9"
 sha3 = "0.9"
 flate2 = "1.0"
 infer = "0.15.0"
-ion-rs = { version = "1.0.0-rc.7", features = ["experimental", "experimental-ion-hash"] }
+ion-rs = { version = "=1.0.0-rc.7", features = ["experimental", "experimental-ion-hash"] }
 tempfile = "3.2.0"
 ion-schema = "0.10.0"
 lowcharts = "0.5.8"

